### DISCUSSION TITLE
"Downcast alphas_cumprod: True"  in  Override Settings?

### DISCUSSION THREAD

**Discussion by pw405** [lshqqytiger/stable-diffusion-webui-directml#406] [General]
After the two recent updates, I've noticed whenever I use the blue arrow to populate my previous txt2img parameters, or use Infinite Image browsing to send an image to Txt2Img, there is a new note added in the Override settings that says:  `Downcast alphas_cumprod: True`. 

![image](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/20309609/fd223b26-8090-461c-9e1e-260eb8d78886)

I have only previously seen this override settings field be populated if I had used Clip Skip on a prior image.  It doesn't seem to be causing any issues, I've just been removing it.  Curious about what it means is all.  Anybody happen to know?

**Comment by Dnozz** [comment]
also curious

**Comment by Zolxys** [comment]
Quote:
1.8.0 (dev: 1.7.0-225) [2024-01-01]( https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/14145 ) - zero terminal SNR noise schedule option
Slightly changes all image generation. The PR changes alphas_cumprod to be never be fp16 unless the backwards compatibility option is enabled. Backwards compatibility option is "Downcast model alphas_cumprod to fp16 before sampling", and it's automatically enabled when restoring parameters from old pictures (as long as they have Version: ... in infotext).
